middleware/__tests__/serverMiddleware.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { Context } from '@nuxt/types/app';
+import sinon from 'sinon';
```

Comment:
Any reason for using sinon instead of [Jest's mock functions](https://jestjs.io/docs/en/mock-functions) ?

Reply:
From what I'm reading, Jest's mocking can do pretty much everything Sinon does. Seems like Sinon is usually used with Mocha, which doesn't have spies (since it takes a more modular approach, which requires you to depend on other things, whereas Jest is an all-in-one testing framework).

Reply:
No reason. I've updated to use jest